File: packages/python-sdk/src/natlas/backends/hf_endpoint.py
"""A Hugging Face Inference Endpoint running N-ATLaS under vLLM.

It speaks the same OpenAI-compatible protocol as ``/serve``. Point
``base_url`` at that endpoint. Speech routes exist only when the endpoint
is the N-ATLAS Kit gateway.
"""

from __future__ import annotations

from natlas.backends.openai_compatible import (
    resolve_openai_compatible_base as resolve_hf_endpoint_base,
)

__all__ = ["resolve_hf_endpoint_base"]

File: packages/js-sdk/src/backends/hf-endpoint.ts
/**
 * A Hugging Face Inference Endpoint running N-ATLaS under vLLM speaks the
 * same OpenAI-compatible protocol as `/serve`. Point `baseURL` at that
 * endpoint. Speech routes exist only when the endpoint is the N-ATLAS Kit
 * gateway; a bare LLM endpoint has no `/v1/audio/transcriptions`.
 */
export { resolveOpenAICompatibleBase as resolveHfEndpointBase } from './openai-compatible.js';
